src/lv_fonts/lv_font_template.c

```diff
@@ -110,10 +111,11 @@ static uint8_t kern_right_class_mapping[] =
         1, 2, 3, 2, 2, 3, 3, 1
 };
 
-/*Kern values between classes*/
+/* Kern values between classes
+ * 12.4 format. (rael_value * 16 is stored)*/
 static uint8_t kern_class_values =
 {
-        LV_FONT_X4_SET(2, 7), LV_FONT_X4_SET(-1, 3), LV_FONT_X4_SET(0, 13), LV_FONT_X4_SET(5, 7)
+        44, 66, 24, 23, 56, 23
 };
```

Comment:
probably should be signed (int8_t)?

Reply:
True.
Updated.

Reply:
`* 12.4 format which needs to scaled with`

=>

`* FP4.4 format which needs to scaled with`

data is FP4.4 here, not 12.4

Reply:
Fixed in 0aeff32